setup.cfg

```diff
@@ -121,4 +121,4 @@ sqlfluff =
     py.typed
 
 [sqlfluff_docs]
-stable_version = 1.4.5
+stable_version = 2.0.0a4
```

Comment:
Is this right, given it would be a `Pre-release` version?

Reply:
Eep, good catch!